src/services/LogFileManager/index.ts

```diff
@@ -206,21 +177,14 @@ class LogFileManager {
         fileData: Uint8Array,
```

Comment:
i meant to say that we can remove `#initDecoder()` as a static method inside the class and just call `resolveDecoderAndFileType()` instead of `#initDecoder()`. then the constructor of LogFileManager can accept the fileTypeInfo as an argument